client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -76,31 +84,87 @@ const PersonalPurchase = ( {
 						<li>{ translate( "You'll see upsells and ads in the Stats page" ) }</li>
 					</ul>
 				) : (
-					<ul className={ `${ COMPONENT_CLASS_NAME }__benefits--included` }>
-						<li>{ translate( 'Instant access to upcoming features' ) }</li>
-						<li>{ translate( 'Priority support' ) }</li>
-						<li>{ translate( 'Ad-free experience' ) }</li>
-					</ul>
+					<>
+						<p>{ translate( 'Benefits:' ) }</p>
+						<ul className={ `${ COMPONENT_CLASS_NAME }__benefits--included` }>
+							<li>{ translate( 'Instant access to upcoming features' ) }</li>
+							<li>{ translate( 'Priority support' ) }</li>
+							<li>{ translate( 'Ad-free experience' ) }</li>
+						</ul>
+					</>
 				) }
 			</div>
 
+			{ subscriptionValue === 0 && (
+				<p>
+					<p>
```

Comment:
We probably don't want to [nest `p` elements](https://stackoverflow.com/questions/9852312/list-of-html5-elements-that-can-be-nested-inside-p-element#9852381) 🙂